translations/harbour-seaprint-de.ts

```diff
@@ -3059,551 +3059,551 @@ auf diesem Drucker</translation>
+        <translation>Briefumschlagsmaschine unbehebbarer Fehler</translation>
     </message>
     <message>
         <source>Envelope Maker Unrecoverable Storage Error</source>
-        <translation type="unfinished"></translation>
+        <translation>Briefumschlagsmaschine unbehebbarer Speicherfehler</translation>
     </message>
     <message>
         <source>Envelope Maker Warming Up</source>
-        <translation type="unfinished"></translation>
+        <translation>Briefumschlagsmaschine wärmt auf</translation>
     </message>
     <message>
         <source>Marker Adjusting Print Quality</source>
-        <translation type="unfinished"></translation>
+        <translation>Kennzeichnung für die Druckqualität einstellen</translation>
     </message>
     <message>
         <source>Marker Cleaner Missing</source>
-        <translation type="unfinished"></translation>
+        <translation>Kennzeichnung Reinigungsmittel fehlt</translation>
     </message>
     <message>
         <source>Marker Developer Almost Empty</source>
-        <translation type="unfinished"></translation>
+        <translation>Kennzeichnung für den Entwickler fast leer</translation>
     </message>
     <message>
         <source>Marker Developer Empty</source>
-        <translation type="unfinished"></translation>
+        <translation>Kennzeichnung für den Entwickler leer</translation>
     </message>
     <message>
         <source>Marker Developer Missing</source>
-        <translation type="unfinished"></translation>
+        <translation>Markierung für den Entwickler fehlt</translation>
     </message>
     <message>
         <source>Marker Fuser Missing</source>
-        <translation type="unfinished"></translation>
+        <translation>Kennzeichnung für die Fixiereinheit fehlt</translation>
     </message>
     <message>
         <source>Marker Fuser Thermistor Failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Kennzeichnung Fixiereinheit Thermistor-Ausfall</translation>
     </message>
     <message>
         <source>Marker Fuser Timing Failure</source>
-        <translation type="unfinished"></translation>
+        <translation>Kennzeichnung Fixiereinheit Messfehler</translation>
     </message>
     <message>
         <source>Marker Ink Almost Empty</source>
-        <translation type="unfinished"></translation>
+        <translation>Kennzeichnung Tinte fast leer</translation>
     </message>
     <message>
         <source>Marker Ink Empty</source>
-        <translation type="unfinished"></translation>
+        <translation>Kennzeichnung Tinte leer</translation>
     </message>
     <message>
         <source>Marker Ink Missing</source>
-        <translation type="unfinished"></translation>
+        <translation>Kennzeichnung Tinte fehlt</translation>
     </message>
     <message>
         <source>Marker Opc Missing</source>
         <translation type="unfinished"></translation>
```

Comment:
Intentional?

Reply:
You mean line 3190? I translate it with        
 <translation>Identification opc missing</translation> I don't know why you don't see it. It is there in the file. By the way: my translation sounds good, but honestly I don't know what OPC is.
If I Searched about it on the Internet, I found out it means Grape seed extract. But I think that is not correct in your App ;)

Reply:
Probably... GitHub is not great at tracking what line i clicked
https://www.scc-inc.com/Newsroom/How-A-Laser-Printer-Works-The-OPC-Drum